react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -84,6 +84,8 @@
     public ReactAztecText(ThemedReactContext reactContext) {
         super(reactContext);
 
+        setGutenbergMode(true);
```

Comment:
Hey @Tug @hypest I have tried to test WPAndroid against this PR and it's crashing with this error: `java.lang.NoSuchMethodError: No virtual method setGutenbergMode(Z)V in class Lorg/wordpress/mobile/ReactNativeAztec/ReactAztecText`

I can see that we are targeting Aztec version `aztecVersion = 'gb-stray-selection-fix-try1'` but when I compare this branch: https://github.com/wordpress-mobile/AztecEditor-Android/tree/gb-stray-selection-fix-try1 with develop it seems they are identical: https://github.com/wordpress-mobile/AztecEditor-Android/compare/gb-stray-selection-fix-try1

I assume that we are missing some commits from that branch?